karma.conf.babel.js

```diff
@@ -0,0 +1,80 @@
+import paths from './paths';
+import statConfig from './webpack-stats';
+import friendlyFormatter from 'eslint-friendly-formatter';
+import exitPlugin from './webpack-exit-plugin';
+import ENV from './ENV';
+
+/**
+ * This config is for running tests on the command line and will fail on errors.
+ * @param {{}} config Karma config object.
+ * @type {{}}
+ */
+export default (config) => {
+  config.set({
+    browsers: ['PhantomJS'],
+    singleRun: !ENV.isDevelopment(),
+    reporters: [ENV.isDevelopment() ? 'mocha' : 'dots'],
+    files: [
+      './node_modules/phantomjs-polyfill/bind-polyfill.js',
+      './test/tests.bundle.js'
+    ],
+    frameworks: [
+      'mocha'
+    ],
+    preprocessors: {
+      '**/*.bundle.js': ['webpack', 'sourcemap']
+    },
+    client: {
+      mocha: {
+        // require: '',
+        reporter: 'html',   // change Karma's debug.html to mocha web reporter
+        ui: 'bdd'
+      }
+    },
+    webpack: {
+      // karma watches the test entry points
+      // (you don't need to specify the entry option)
+      // webpack watches dependencies
+      devtool: 'inline-source-map',
```

Comment:
+1 Let's keep this PR about the babel upgrade and add issues for anything else.  The template updates later should also cover this.